src/activities/menu/ConfigurationItem.qml

```diff
@@ -404,6 +413,58 @@ Item {
                 }
             }
         }
+        
+        Flow {
```

Comment:
So, what could be the optimum choice keeping in mind the advantages and disadvantages of both the methods? Should we change it the wordset way and do it as done in checkWordset() or keep it the way it is currently (letting the user decide when to download and update when necessary and not just keeping the download for start)?